README.md

```diff
@@ -19,14 +19,50 @@ go get gopkg.in/DataDog/dd-trace-go.v1/contrib/gorilla/mux
 
 If you installed more packages than you intended, you can use `go mod tidy` to remove any unused packages.
 
-Requires:
+### Documentation
+
+The API is documented on [godoc](https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace) as well as Datadog's official documentation for [Tracing Go Applications](https://docs.datadoghq.com/tracing/setup/go/) and the [Continuous Go Profiler](https://docs.datadoghq.com/tracing/profiler/enabling/go). If you are migrating from an older version of the tracer (e.g. 0.6.x) you may also find the [migration document](https://github.com/DataDog/dd-trace-go/blob/v1/MIGRATING.md) we've put together helpful.
+
+### Support Policy
+
+Datadog APM for go is built upon dependencies defined in specific versions of the host operating system, go releases, and the Datadog Agent/API. For go the two latest releases are [GA](#support-ga) supported and the version before that is in [Maintenance](#support-maintenance). We do make efforts to support older releases, but generally these releases are considered [Legacy](#support-legacy). This library only officially supports [first class ports](https://github.com/golang/go/wiki/PortingPolicy#first-class-ports) of go.
+
+| **Level**                                              | **Support provided**                                                                                                                                                          |
+|--------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-ga">General Availability (GA)</span> |  Full implementation of all features. Full support for new features, bug & security fixes.                                                                                    |
```

Comment:
Thanks for writing this up Andrew! I'm curious about the meaning of the different support levels. In the context of which Go versions we support, can we have anything other than GA in practice? If we use a new standard library API or language feature from a new Go version, the library won't compile at all for older Go versions.

Reply:
Good question, I think generally speaking that all of the versions we "Support" are really "GA". However I like the idea of limiting our support for the no-longer-supported-by-Go oldest version. The best I can come up with is this hypothetical:

Suppose we're at the point where 1.17 is in maintenance and 1.18 and 1.19 are GA. We run into a really useful library that will allow us to provide a new tracer feature. We would be OK to add that feature/library behind some protected build guards so that users of 1.18/1.19 can immediately begin to use the new feature and users of 1.17 could still safely upgrade to get access to things like bug fixes etc, but wouldn't have access to that new feature.

This is a pretty synthetic example and if it seems like something we don't want to do / need to do I'm also happy dropping the term "maintenance" and just saying we support all 3 versions as GA. For the synthetic example it would mean either waiting for a new release before creating the new feature or writing the feature in a way that supports all the versions.